Branding Update Combined (#4)
openstudiocoalition/openstudiocoalition.github.io: markdown_source/docs/about/roadmap.md

<h1>Roadmap Features</h1>
After the v1.0.0 NREL-provided software distribution, the OpenStudio Application will continue to be supported with open source contributions by The OpenStudio Coalition and members of the software community. Plans include support for releasing new versions of the OpenStudio Application that stay aligned with scheduled releases of the OpenStudio SDK, with minimal or no changes to the user interface. Additionally, critical bugfix releases may be provided, on an as-needed basis. 

The OpenStudio SDK, OpenStudio Command Line Interface (CLI), OpenStudio Server, OpenStudio Standards, OpenStudio Measures, Floorspace.JS Widget (Floorspace), Building Component Library (BCL), and Parametric Analysis Tool (PAT) will be unaffected. However, these tools will continue to be developed, distributed, and maintained by NREL. See [https://www.openstudio.net/new-future-for-openstudio-application](https://www.openstudio.net/new-future-for-openstudio-application) for details. 

## Partnerships and Open Source Software Contribution Opportunities

> Effective early May 2020, NREL will no longer distribute the OpenStudio Application and the SketchUp Plugin. The final NREL software distribution of the OpenStudio Application, along with any non-NREL future releases, will be available at [https://github.com/NREL/OpenStudioApplication/releases](https://github.com/NREL/OpenStudioApplication/releases). Companies, individual contributors, and organizations that are interested in providing development contributions, support and maintenance activities for the OpenStudio Application, including new features, defect fixes and enhancements, are encouraged to contact OSC@openstudiocoalition.org.

Author: ```suggestion
As of May 2020, NREL no longer distributes the OpenStudio Application and the SketchUp Plugin. The final NREL software distribution of the OpenStudio Application, along with any non-NREL future releases, remains available at [https://github.com/NREL/OpenStudioApplication/releases](https://github.com/NREL/OpenStudioApplication/releases). 

After the final release from NREL, the OpenStudio Coalition has continued to maintain the OpenStudio Application and SketchUp Plugin.  The OpenStudio Coalition aims to organize a healthy community of OpenStudio users with interest in continued maintenance of the OpenStudio Application and SketchUp Plugin as free and open-source tools. Companies, individual contributors, and organizations that are interested in providing development contributions, support and maintenance activities for the OpenStudio Application or SketchUp Application, including new features, defect fixes and enhancements, are encouraged to contact OSC@openstudiocoalition.org.
```

Author: @marccosta we want to bring this up to the current day

> 

Author: ```suggestion
## Other ways to help

Non-monetary in-kind support is also appreciated and includes:
- Testing of release candidates and new features
- Direct code contributions for bug fixes and new features
- Documentation or training material contributions
```

> ## List of Features Needing Funding

Author: ```suggestion
## List of Features Seeking Funding
```

###Distributed Energy Resources
   - Expand capabilities for the app to model solar, storage, electric vehicles, and heat pumps (water and space conditioning)

###BCL Expansion

>    - Develop and provide guidance on building component library (BCL) measures to allow for decarbonization analysis
>    - Develop measures for REopt and/or SAM to allows energy efficiency measures to be modeled along with DERs
>    - Integrate U.S. DOE National Lab demand flexibility algorithms via BCL measures 

Author: ```suggestion
   - Develop and support building component library (BCL) measures to allow for decarbonization analysis.  This could include measures which integrate REopt, SAM, demand flexibility, or other DER focused tools.
```

Author: @marccosta I tried to edit this to sound less DOE/NREL specific

   
###Interface
   - Modernize UI design

>    - Language translation localization for EFIGS and beyond

Author: @marccosta what is EFIGS?

   
###Wizards
   - Gather feedback from user community on what wizards or simplified use cases are most helpful
   
###Compliance Capabilities

>    - Complete the testing and documentation required for California Title 24 software compliance

Author: ```suggestion
   - Complete the testing and documentation required for California Title 24 software compliance
   - Create and maintain OpenStudio Application interfaces and reporting measures for ASHRAE 229 comparison.
```

>    
> 

Author: ```suggestion
```
